modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/bnd.bnd

```diff
@@ -5,7 +5,7 @@ Export-Package: ${package}.constants
 
 -includeresource:\
 	@com.liferay.util.bridges-[0-9]*.jar!/com/liferay/util/bridges/freemarker/FreeMarkerPortlet.class,\
-	@com.liferay.util.taglib-[0-9]*.jar!/META-INF/*.tld#if (${buildType} != "gradle")
+	@com.liferay.util.taglib-[0-9\.]*.jar!/META-INF/*.tld
```

Comment:
Hey @lawrence-lee  you removed the #if {} statement here.  Was that intentional?

Reply:
Nevermind!  I see you did it in a followup commit.